README.md

```diff
@@ -32,17 +32,20 @@ Standards and specifications
 Dev tooling
 -----------
 
-**[dp tool](cmd/README.md)**
+**[dp command line tool](https://github.com/ONSdigital/dp-cli)**
 
-A command line utility to help make working in Digital Publishing simpler.
+A command line utility to help make working in Digital Publishing simpler. Now hosted in its own repository
```

Comment:
```suggestion
A command line utility to assist work in Digital Publishing. See its repository
```
"now hosted" is a short-term statement
"help make working ... simpler" is excessive
(I would also be tempted to hyphenate "command-line")